lib/menu.rb

```diff
@@ -0,0 +1,31 @@
+class Menu
+
+  def initialize
+    @menu_list = {:steak => 15, :chips => 5, :pizza =>7,
+    :lasagna =>12, :chicken => 10, :sushi => 14}
+  end
+
+  def food
+    menu_string = "\nMenu\n"
+    @menu_list.each {|key,value|
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)